src/Workspaces/CSharp/Portable/CodeStyle/UsingPlacementPreference.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeStyle
+{
+    internal enum UsingPlacementPreference
+    {
+        ///<summary>
+        /// No preference whether usings should be placed inside or outside namespaces.
+        ///</summary>
+        NoPreference,
+
+        ///<summary>
+        /// Prefer usings placed inside namespaces.
+        ///</summary>
+        InsideNamespace,
+
+        ///<summary>
+        /// Prefer usings placed outside namespaces.
+        ///</summary>
+        OutsideNamespace
```

Comment:
better names might be 'CompilationLevel' and 'NamespaceLevel'.  You may not have a namespace in your code, so saying 'OutputNamespace' doesn't really make a ton of sense to me.

Reply:
With only two values, this could potentially also be a bool.  though i' mnot opposed to an enum.

Reply:
I don't see the suggested alternatives as a clear improvement over the current names. The current names align with the documentation described by both StyleCop Classic and StyleCop Analyzers, so it seems consistent with what users would likely expect today.